docs/user_manual.md

```diff
@@ -0,0 +1,68 @@
+# User manual for EIT dashboard
+
+Note 1: If you have not installed the dashboard yet, please refer to our [README](../README.md) on instructions how to do
+that.
+
+Note 2: While the dashboard is rendered in a browser window, you are not working online. All data that you load into the
+dashboard remains local on your machine.
+
+Note 3: The Dashboard, this manual, and the (back end) software it is built around are works in progress. The images and
+interaction procedures may be slightly off. Please bear with us.
+
+## LOAD page
+
+The dashboard should open on the **LOAD** page, but in case this does not happen, navigate there by clicking **LOAD** at
+the top of the page. You should see something like this:
+
+<kbd>
+<img src=images/load_entry_point.png width="1200px">
+</kbd>
+
+Click on the dropdown menu to select the format of data to load, then click on the `Select Files` button. From here you
+can navigate to the data and select it. Hit `Confirm` to pre-load and pre-view the selected data.
+
```

Comment:
Maybe this needs a section on what type of data files can be loaded. For Sentec those are *.zri files, for Dräger those are *.bin files exported from the PV500 software, and for Timpel those are *.txt files exported from ... @ajonkman, what did you export those with?